README.md

```diff
@@ -6,46 +6,82 @@
 
 # FlutterBleLib
 
-A library for all your Bluetooth Low Energy needs in Flutter. Internally utilises Polidea's 
-[MultiPlatformBleAdapter](https://github.com/Polidea/MultiPlatformBleAdapter), 
+A library for all your Bluetooth Low Energy needs in Flutter. Internally utilises Polidea's
+[MultiPlatformBleAdapter](https://github.com/Polidea/MultiPlatformBleAdapter),
 which runs on [RxAndroidBle](https://github.com/Polidea/RxAndroidBle)
- and [RxBluetoothKit](https://github.com/Polidea/RxBluetoothKit).
+and [RxBluetoothKit](https://github.com/Polidea/RxBluetoothKit).
 
 ## BLE Simulator
-This library supports [BLEmulator](https://github.com/Polidea/blemulator_flutter), the BLE simulator. 
+
+This library supports [BLEmulator](https://github.com/Polidea/blemulator_flutter), the BLE simulator.
 The simulation allows one to develop without physical smartphone or BLE peripheral and use one's production BLE–related code in automated testing.
 
+## Installation
+
+To use this plugin, add `flutter_ble_lib` as a [dependency in your pubspec.yaml file](https://flutter.dev/docs/development/packages-and-plugins/using-packages).
+
+### Android
+
+Set `minSDKVersion` in `[project]/android/app/build.gradle` file to 18.
+
+```gradle
+defaultConfig {
+  ...
+  minSdkVersion 18
+  ...
+}
+```
+
+Support for Bluetooth Low Energy has been added in API 18, hence the library requires `minSDKVersion` to be set to 18. If BLE is not core to your application, [you can override it](https://stackoverflow.com/questions/27335889/how-do-i-ignore-minsdkversion-of-library-in-android-studio) and handle support detection in your code.
+
+**Notice:** You don't need to add any additional permissions related to BLE because they are already declared in our manifest. However, you still need to request location permission at runtime, which you can implement with [permission_handler](https://pub.dev/packages/permission_handler) package. See [our example](https://github.com/Polidea/FlutterBleLib/tree/master/example) for reference.
```

Comment:
```suggestion
**Notice:** You don't need to add any permissions related to BLE to the AndroidManifest.xml, because they are already declared in the library's native module. However, you still need to request `ACCESS_FINE_LOCATION` permission at runtime to be able to scan for peripheral. See [example's code](https://github.com/Polidea/FlutterBleLib/blob/develop/example/lib/devices_list/devices_bloc.dart#L80) and [example's pubspec](https://github.com/Polidea/FlutterBleLib/blob/develop/example/pubspec.yaml#L20).
```